webdriver_wharf/interactions.py

```diff
@@ -24,15 +24,14 @@
     print 'WEBDRIVER_WHARF_START_TIMEOUT must be an integer, defaulting to 60'
     start_timeout = 60
 
-# TODO: Making these configurable would be good
-# lowest port for webdriver binding
-_wd_port_start = 4900
-# Offsets for finding the other ports
-_vnc_port_offset = 5900 - _wd_port_start
-_http_port_offset = 6900 - _wd_port_start
+
+PORT_SSH = u'22/tcp'
+PORT_HTTP = u'80/tcp'
+PORT_WEBDRIVER = u'4444/tcp'
+PORT_VNC = u'5999/tcp'
 
 # docker client is localhost only for now
-client = AutoVersionClient(timeout=120)
+client = docker.from_env(timeout=120, version='auto')
```

Comment:
It's possible I was using an outdated lib, but upgrading it changes the exception to this, which doesn't seem better:
```python
Traceback (most recent call last):
  File "/home/semyers/ve/webdriver-wharf/bin/webdriver-wharf", line 6, in <module>
    from webdriver_wharf.cli import main
  File "/home/semyers/ve/webdriver-wharf/webdriver-wharf/webdriver_wharf/cli.py", line 5, in <module>
    from webdriver_wharf import app, logging_init
  File "/home/semyers/ve/webdriver-wharf/webdriver-wharf/webdriver_wharf/app.py", line 13, in <module>
    from webdriver_wharf import db, interactions, lock
  File "/home/semyers/ve/webdriver-wharf/webdriver-wharf/webdriver_wharf/interactions.py", line 17, in <module>
    from docker import AutoVersionClient, errors
ImportError: cannot import name AutoVersionClient
```

Is the requirements file still right?